modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/http4s/SeqexecCommandRoutes.scala

```diff
@@ -73,33 +77,35 @@ class SeqexecCommandRoutes(auth: AuthenticationService, inputQueue: engine.Event
       } yield resp
 
 
-    case req @ POST -> Root / "conditions" =>
-      req.decode[Conditions] (conditions =>
+    case req @ POST -> Root / "conditions" as _ =>
+      req.req.decode[Conditions] (conditions =>
```

Comment:
If you have authentication yes. You don't get a plain request (possibly the tame of the variable should change) and you get an `AuthedRequest` instead that contains the original `Request` in the `req` variable